pyo3-ffi-check/build.rs

```diff
@@ -1,6 +1,24 @@
 use std::env;
 use std::path::PathBuf;
 
+#[derive(Debug)]
+struct ParseCallbacks;
+
+impl bindgen::callbacks::ParseCallbacks for ParseCallbacks {
```

Comment:
Thanks @davidhewitt for the suggestion, this worked like a charm. I don't see a callback API to override struct member names, so I had to leave the hack in the macro for the `ob_refcnt` field, but this lets us generate the names that pyo3-ffi expects for the type names I override below.